modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1054,9 +1077,18 @@ EFAElement3D::createChild(const std::set<EFAElement *> & CrackTipElements,
         EFAPoint p(0.0, 0.0, 0.0);
         p = _local_node_coor[j];
         EFAPoint origin_to_point = p - orig;
+
+        bool use_temp = false;
```

Comment:
This variable could use a better name to improve readability. Maybe `node_outside_fragment`?